config/locales/frontend.en.yml

```diff
@@ -0,0 +1,53 @@
+en:
+  frontend:
+    load: "Load more"
+    success:
+      title: "Success!"
+    error:
+      title: "Uh-oh..."
+      message: "An error occurred, a developer should check the console for details"
+    subscription:
+      subscribe: "Successfully subscribed."
+      unsubscribe: "Successfully unsubscribed."
+      fail:
+        subscribe: "Failed to subscribe to answer."
+        unsubscribe: "Failed to unsubscribe from answer."
+    list:
+      confirm:
+        title: "Really delete this list?"
+        text: "You will not be able to recover this list."
+    destroy_question:
+      confirm:
+        title: "Are you sure?"
+        text: "The question will be moved back to your inbox, but it won't delete any posts to social media."
+    destroy_comment:
+      confirm:
+        title: "Really delete?"
```

Comment:
We should replace all of these "Really delete?" strings with "Are you sure?". They read a bit weird